ext_mongodb.php

```diff
@@ -412,10 +412,10 @@ public function __debugInfo() : Array
 		return [
 			'query' => (object) $this->query['query'],
 			'selector' => array_key_exists('fields', $this->query) ? (object) $this->query['fields'] : NULL,
-			'flags' => $this->query['flags'],
-			'skip' => $this->query['skip'],
-			'limit' => $this->query['limit'],
-			'batch_size' => $this->query['batchSize'],
+			'flags' => array_key_exists('flags', $this->query) ? $this->query['flags'] : 0,
+			'skip' => array_key_exists('skip', $this->query) ? $this->query['skip'] : 0,
+			'limit' => array_key_exists('limit', $this->query) ? $this->query['limit'] : 0,
+			'batch_size' => array_key_exists('batchSize', $this->query) ? $this->query['batchSize'] : 0,
```

Comment:
Assuming https://github.com/mongodb/mongo-hhvm-driver/pull/101/files#diff-511ccdfa2d2a7f6f84733ed3cc22a2e6R371 is the only place where you initialize these values, is there any reason not to simply use `isset()` here?

Also, I'm not sure if you're supporting HHVM versions below 3.11, but [null coalesce](https://github.com/facebook/hhvm/commit/fcf8d4dea785cdbcba02baa7a0d94020d14a9a10) is also an option.